src/main/java/me/totalfreedom/totalfreedommod/command/Command_smite.java

```diff
@@ -38,22 +39,23 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             return true;
         }
 
-        smite(player, reason);
+        smite(player, sender, reason);
         return true;
     }
 
-    public static void smite(Player player)
+    public static void smite(Player player, CommandSender sender)
```

Comment:
Pretty sure some other command/listener uses this method and that's why it's there. If you haven't updated the arg in the other class, it won't compile. Correct me if I'm wrong.